packages/turbo-codemod/src/transforms/clean-globs.ts

```diff
@@ -0,0 +1,106 @@
+import { TransformerArgs } from "../types";
+import type { Schema as TurboJsonSchema } from "@turbo/types";
+import { TransformerResults } from "../runner";
+import path from "path";
+import fs from "fs-extra";
+import getTransformerHelpers from "../utils/getTransformerHelpers";
+import { getTurboConfigs } from "@turbo/utils";
+import Logger from "../utils/logger";
+
+// transformer details
+const TRANSFORMER = "clean-globs";
+const DESCRIPTION =
+  "Automatically clean up invalid globs from your &#x60;turbo.json&#x60; file";
```

Comment:
Is there a reason for using an accent over just a `'`? If we continue going with the accent I'd much rather have it in it's own constant and use it in the construction of `DESCRIPTION`

Reply:
This was the default output from `plop`. Maybe you can weigh in @tknickman ?

Reply:
Hmmm it's probably an issue with how I wrote the template. I might need an escaping helper when writing the output. I would just use the `'`